presto-spi/src/test/java/io/prestosql/spi/testing/InterfaceTestUtils.java

```diff
@@ -64,23 +64,11 @@ private InterfaceTestUtils() {}
                     }));
 
             try {
-                if (!defines(forwardingInstance, actualMethod)) {
```

Comment:
Please rename method from `assertProperForwardingMethodsAreCalled` to `assertForwardingClass` and maybe add a javadoc that explains what is a forwarding class.

Reply:
this commit is just a revert.